tests/foreman/cli/test_http_proxy.py

```diff
@@ -272,6 +272,13 @@ def test_positive_assign_http_proxy_to_products(module_org, module_target_sat):
         }
     )
     assert 'Product proxy updated' in res
+    module_target_sat.wait_for_tasks(
```

Comment:
I beleive this function returns `List of sat.api.ForemanTask entities`, would it be possible to use assersion on return value for this call?